scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -4442,3 +4489,31 @@ function loaded() {
         EventBridge.emitWebEvent(JSON.stringify({ type: 'propertiesPageReady' }));
     }, 1000);
 }
+
+function showOnTheSamePage(entityType) {
+    var i;
```

Comment:
this variable is only used in the for loop as a iterator, please define it there